k8s/coordinator/development/cert-volume-mount.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coordinator-deployment
+spec:
+  template:
+    spec:
+      volumes:
+        - name: tls-certificate
+          secret:
+            secretName: dev-coordinator
+            items:
+              - key: tls.key
+                path: tls.key
+                mode: 0400
+              - key: tls.crt
+                path: tls.pem
+                mode: 0444
```

Comment:
Why not `400`?

Reply:
The certificate file itself can be world readable without any problems. I just followed what's common practice by having it be `0444` (it's actually usually `0644`, but since I already mount the secret as a read-only file, I went the extra mile and made it be `0444` to reflect that).